tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/AttendanceStatus.java

```diff
@@ -0,0 +1,67 @@
+package com.linagora.tmail.james.jmap;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import jakarta.mail.Flags;
+
+import org.apache.james.mailbox.FlagsBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Preconditions;
+
+public enum AttendanceStatus {
+    Accepted("$accepted"),
+    Declined("$rejected"),
+    Tentative("$tentativelyaccepted"),
+    NeedsAction("$needs-action");
```

Comment:
We removed the spec-defined $Delegated ? 

Why?

Reply:
Delegated is not in use now. It would additionally require some extra handling because, in addition to the delegated flag, we need to keep track of whom we are delegating our event attendance status, so I'm thinking of just removing it from the enum.